chains/orchestrator-relays/runtime/starlight/src/lib.rs

```diff
@@ -0,0 +1,3813 @@
+}
+
+pub const OPEN_STREAM_HOLD_AMOUNT: u32 = 253;
+parameter_types! {
+    // 1 entry, storing 253 bytes on-chain in the worst case
+    pub const OpenStreamHoldAmount: Balance = deposit(1, OPEN_STREAM_HOLD_AMOUNT);
+}
+
+impl pallet_stream_payment::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type StreamId = tp_stream_payment_common::StreamId;
+    type TimeUnit = tp_stream_payment_common::TimeUnit;
+    type Balance = Balance;
+    type AssetId = tp_stream_payment_common::AssetId;
+    type AssetsManager = tp_stream_payment_common::AssetsManager<Runtime>;
+    type Currency = Balances;
+    type OpenStreamHoldAmount = OpenStreamHoldAmount;
+    type RuntimeHoldReason = RuntimeHoldReason;
+    type TimeProvider = tp_stream_payment_common::TimeProvider<Runtime>;
+    type WeightInfo = weights::pallet_stream_payment::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+    pub const ProfileDepositBaseFee: Balance = STORAGE_ITEM_FEE;
+    pub const ProfileDepositByteFee: Balance = STORAGE_BYTE_FEE;
+    #[derive(Clone)]
+    pub const MaxAssignmentsPerParaId: u32 = 10;
+    #[derive(Clone)]
+    pub const MaxNodeUrlLen: u32 = 200;
+}
+
+impl pallet_data_preservers::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeHoldReason = RuntimeHoldReason;
+    type Currency = Balances;
+    type WeightInfo = weights::pallet_data_preservers::SubstrateWeight<Runtime>;
+
+    type ProfileId = u64;
+    type ProfileDeposit = tp_traits::BytesDeposit<ProfileDepositBaseFee, ProfileDepositByteFee>;
+    type AssignmentProcessor = tp_data_preservers_common::AssignmentProcessor<Runtime>;
+
+    type AssignmentOrigin = pallet_registrar::EnsureSignedByManager<Runtime>;
+    type ForceSetProfileOrigin = EnsureRoot<AccountId>;
+
+    type MaxAssignmentsPerParaId = MaxAssignmentsPerParaId;
+    type MaxNodeUrlLen = MaxNodeUrlLen;
+    type MaxParaIdsVecLen = MaxLengthParaIds;
+}
+
+parameter_types! {
+    pub StarlightBondAccount: AccountId32 = PalletId(*b"StarBond").into_account_truncating();
+    pub PendingRewardsAccount: AccountId32 = PalletId(*b"PENDREWD").into_account_truncating();
+
+    // 30% for starlight bond, so 70% for staking
+    pub const RewardsPortion: Perbill = Perbill::from_percent(70);
+}
+
+// We want a global annual inflation rate of 10%.
+// It is compounded throught era inflations, which itself is split between:
+// - Inflation for collators per block
+// - Inflation for validators per era
+// Computation is implemented in tests/inflation_rates.rs, with a test ensuring values from the
+// runtime match the formulas. We write the results as constants here to ensure we don't perform
+// computations at runtime.
+prod_or_fast_parameter_types! {
+    pub const CollatorsInflationRatePerBlock: Perbill = { prod: Perbill::from_parts(9), fast: Perbill::from_parts(9) };
+    pub const ValidatorsInflationRatePerEra: Perbill = { prod: Perbill::from_parts(130570), fast: Perbill::from_parts(272) };
```

Comment:
@nanocryk is this right considering one era is one day (instead of 6 hours?) in starlight? probabyl not right no? how can we calculate this?

Reply:
To compute the correct value, the test checking the rates must be modified with the desired durations, and then it'll complain the runtime values don't match the computed ones :)